R/utils.R

```diff
@@ -1,44 +1,41 @@
-#' Removes curly brackets from string
+#' Removes leading and trailing curly brackets from character
+#' string and removes indentation of remaining contents
 #'
 #' @description `r lifecycle::badge("stable")`
 #' @param x (`character`)\cr
-#'  string with code chunk
 #'
-#' @return The string without curly braces
+#' @return Character string without curly braces
 #' @keywords internal
-#'
 remove_enclosing_curly_braces <- function(x) {
   checkmate::assert_character(x)
   if (length(x) == 0) {
-    x
-  } else {
-    txt <- unlist(strsplit(x, "\n", fixed = TRUE))
-    if (grepl("^\\{[[:space:]]*", txt[1]) && grepl("^\\}[[:space:]]*", utils::tail(txt, 1))) {
-      txt2 <- txt[-c(1, length(txt))]
+    return(x)
+  }
+
+  open_bracket_then_spaces <- "^\\{[[:space:]]*"
+  close_bracket_then_spaces <- "^\\}[[:space:]]*"
+  blank_line <- "^[[:space:]]*$"
+  four_spaces_at_start_of_line <- "^[[:space:]]{4}"
```

Comment:
```suggestion
  open_bracket_then_spaces <- "^\\{[[:blank:]]*"
  close_bracket_then_spaces <- "^\\}[[:blank:]]*"
  blank_line <- "^[[:blank:]]*$"
  four_spaces_at_start_of_line <- "^[[:blank:]]{4}"
```

should be the same but more direct.

Reply:
done